.github/workflows/pr-main-awesome.yml

```diff
@@ -0,0 +1,56 @@
+name: pr-main-awesome
```

Comment:
Please fix the name and the file name as well.

Reply:
done